### ISSUE TITLE
File options incorrectly retained when excluding all descriptors.

### ISSUE THREAD

**Issue by JakeWharton** [square/wire#497]
```proto
message Foo {
}

extend google.protobuf.FileOptions {
  optional Foo foo = 25000;
}
```
with `excludes("google.protobuf.*")` will incorrectly retain `Foo` in the final `Schema`.

**Comment by JakeWharton** [comment]
This may be more nuanced. Having trouble writing a test case.

**Comment by JakeWharton** [comment]
Fixed by #500.
